pypeit/core/trace.py

```diff
@@ -607,9 +608,9 @@ def follow_centroid(flux, start_row, start_cen, ivar=None, bpm=None, fwgt=None,
             uses the DISCONTINUOUS flag.
 
     Returns:
-        Three numpy arrays are returned: the optimized center, an
-        estimate of the error, and a bad-pixel mask (masked values
-        are True).
+        tuple: Three numpy arrays are returned: the optimized center, an
```

Comment:
remove the second `:`. Maybe substitute with `. They are the optimized center (...)`

Reply:
done